quickstarts/Function_calling.ipynb

```diff
@@ -749,7 +824,17 @@
         "- The [response](https://ai.google.dev/api/python/google/generativeai/protos/GenerateContentResponse) may contain a [genai.protos.FunctionCall](https://ai.google.dev/api/python/google/generativeai/protos/FunctionCall), in `response.candidates[0].contents.parts[0]`.\n",
         "- if `enable_automatic_function_calling` is set the [genai.ChatSession](https://ai.google.dev/api/python/google/generativeai/ChatSession) executes the call, and sends back the [genai.protos.FunctionResponse](https://ai.google.dev/api/python/google/generativeai/protos/FunctionResponse).\n",
         "- In response to a [FunctionCall](https://ai.google.dev/api/python/google/generativeai/protos/FunctionCall) the model always expects a [FunctionResponse](https://ai.google.dev/api/python/google/generativeai/protos/FunctionResponse).\n",
-        "- If you reply manually using [chat.send_message](https://ai.google.dev/api/python/google/generativeai/ChatSession#send_message) or [model.generate_content](https://ai.google.dev/api/python/google/generativeai/GenerativeModel#generate_content) remember thart the API is stateless you have to send the whole conversation history (a list of [content](https://ai.google.dev/api/python/google/generativeai/protos/Content) objects), not just the last one containing the `FunctionResponse`."
+        "- If you reply manually using [chat.send_message](https://ai.google.dev/api/python/google/generativeai/ChatSession#send_message) or [model.generate_content](https://ai.google.dev/api/python/google/generativeai/GenerativeModel#generate_content) remember thart the API is stateless you have to send the whole conversation history (a list of [content](https://ai.google.dev/api/python/google/generativeai/protos/Content) objects), not just the last one containing the `FunctionResponse`.\n",
```

Comment:
These API links don't work any more. You don't have to change the old ones, but don't add new links that don't work.

(by "don't work", I mean that we redirect anything from `/api/python/*` to the same page - a better experience would be to provide a link into the [specific pages on GitHub](https://github.com/google-gemini/generative-ai-python/blob/main/docs/api/google/generativeai/protos/Content.md), or to the [API reference](https://ai.google.dev/api/generate-content#request-body) for non-Python-specifics)

Reply:
Shoud I change the links to https://github.com/google-gemini/generative-ai-python/blob/main/docs/api/google/generativeai/ChatSession.md#send_message for example?

Reply:
Yeah that works. I prefer using the website where we can but for specific symbols that makes sense.